config/kraft/broker.properties

```diff
@@ -31,22 +31,23 @@ controller.quorum.voters=1@localhost:9093
 
 ############################# Socket Server Settings #############################
 
-# The address the socket server listens on. It will get the value returned from
-# java.net.InetAddress.getCanonicalHostName() if not configured.
+# The address the socket server listens on. If not configured, the host name will get from
+# java.net.InetAddress.getCanonicalHostName(), with PLAINTEXT listener name, and port 9092.
 #   FORMAT:
 #     listeners = listener_name://host_name:port
 #   EXAMPLE:
 #     listeners = PLAINTEXT://your.host.name:9092
 listeners=PLAINTEXT://localhost:9092
+
+# Name of listener used for communication between brokers.
 inter.broker.listener.name=PLAINTEXT
 
-# Hostname and port the broker will advertise to producers and consumers. If not set,
-# it uses the value for "listeners" if configured.  Otherwise, it will use the value
-# returned from java.net.InetAddress.getCanonicalHostName().
+# Listener name, hostname and port the broker will advertise to producers and consumers.
+# If not set, it uses the value for "listeners".
 advertised.listeners=PLAINTEXT://localhost:9092
 
-# Listener, host name, and port for the controller to advertise to the brokers. If
-# this server is a controller, this listener must be configured.
+# A comma-separated list of the names of the listeners used by the controller.
+# This is required if running in KRaft mode.
```

Comment:
On a broker node (which this properties file corresponds to) any listener beyond the first one is ignored.  So it would be good to either append a statement to that effect here.